fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -458,7 +459,10 @@ private fun TranslationsDialogHeader(
 
         if (FxNimbus.features.translations.value().pageSettingsEnabled) {
             IconButton(
-                onClick = { onSettingClicked() },
+                onClick = {
+                    Events.translationsAction.record(Events.TranslationsActionExtra("page_settings"))
```

Comment:
The same idea like here https://github.com/mozilla-mobile/firefox-android/pull/5942/files#r1517587430

Reply:
Updated, ty!